<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>421</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayou">
   <item>
    <widget class="QLabel" name="widgetTitle">
     <property name="text">
      <string>Image Processing Options</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="openingsTitleLayout">
     <item>
      <widget class="QLabel" name="openingsTitle">
       <property name="text">
        <string>Openings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="openingsInfoButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLCDNumber" name="openingsLCD">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="openingsSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dilationsTitleLayout">
     <item>
      <widget class="QLabel" name="dilationsTitle">
       <property name="text">
        <string>Dilations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="dilationsInfoButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLCDNumber" name="dilationsLCD"/>
   </item>
   <item>
    <widget class="QSlider" name="dilationsSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="thresholdTitleLayout">
     <item>
      <widget class="QLabel" name="thresholdTitle">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="thresholdInfoButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLCDNumber" name="thresholdLCD"/>
   </item>
   <item>
    <widget class="QSlider" name="thresholdSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="restoreDefaultsButton">
     <property name="text">
      <string>Restore Defaults</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
